tensorflow/core/kernels/mkl/mkl_conv_grad_input_ops.cc

```diff
@@ -675,24 +675,36 @@ class MklConvCustomBackpropInputOp
           .TypeConstraint<T>("T")                                \
           .Label(mkl_op_registry::kMklLayoutDependentOpLabel),   \
       MklConvCustomBackpropInputOp<CPUDevice, T, false, false>); \
+  REGISTER_KERNEL_BUILDER(                                       \
+      Name("_MklConv3DBackpropInputV2")                          \
+          .Device(DEVICE_CPU)                                    \
+          .TypeConstraint<T>("T")                                \
+          .Label(mkl_op_registry::kMklLayoutDependentOpLabel),   \
+      MklConvCustomBackpropInputOp<CPUDevice, T, false, false>); \
+  REGISTER_KERNEL_BUILDER(                                       \
+      Name("_MklDepthwiseConv2dNativeBackpropInput")             \
```

Comment:
Actually Mkl op name depends on the original non-mkl op. The graph rewrite code would just add a prefix to the original op name, so if I change 'd' to 'D', it won't work. Here is the original op name https://github.com/tensorflow/tensorflow/blob/f96903abead60301aff36739cb145624d7f5576b/tensorflow/core/ops/nn_ops.cc#L586

Reply:
True, because we are just appending `_Mkl` prefix. I might change the original op name (and also the corresponding MKL op name) some time later then. Thank you!